SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -152,6 +152,12 @@ Interest InterestFactory::CreateServerWorkerInterest(const UAbstractLBStrategy*
 	ServerQuery.Constraint.ComponentConstraint = SpatialConstants::WORKER_COMPONENT_ID;
 	AddComponentQueryPairToInterestComponent(ServerInterest, SpatialConstants::POSITION_COMPONENT_ID, ServerQuery);
 
+	// Add a self query to ensure we see the well known entity tag.
+	Query AuthoritySelfQuery = {};
+	AuthoritySelfQuery.ResultComponentIds = { SpatialConstants::GDK_KNOWN_ENTITY_TAG_COMPONENT_ID };
```

Comment:
Is there anything else that we should be interested in if we are the authoritative worker that we're receiving implicitly through another query that may not hold?

Reply:
I can check the other components on the worker entity and see where they are used. I don't think there are more but I haven't really looked into it.

Reply:
It looks like it only has standard components and presence for some reason I'm not really sure about. It doesn't look we do anything with else as a result of authority over the entity without requiring authority over that specific component.